examples/PACKAGES/mbx/1024h2o/in.mbx_h2o

```diff
@@ -0,0 +1,58 @@
+processors      * * * map xyz
+
+units           real
+atom_style      full
+boundary        p p p
+
+read_data       initial.data 
+#read_restart    restart.old
+
+pair_style      mbx 9.0
+pair_modify     mix arithmetic
+
+bond_style      none
+angle_style     none
+dihedral_style  none
+improper_style  none
+
+pair_coeff      * * 1 h2o 1 2 2 json mbx.json
+
+neighbor        2.0 bin
+neigh_modify    every 1 delay 10
+
+timestep        0.5
+
+compute         mbx all pair mbx
+variable        e1bpip    equal c_mbx[1]
+variable        e2bpip    equal c_mbx[2]
+variable        e3bpip    equal c_mbx[3]
+variable        e4bpip    equal c_mbx[4]
+variable        edisp     equal c_mbx[5]
+variable        ebuck     equal c_mbx[6]
+variable        eperm     equal c_mbx[7]
+variable        eind      equal c_mbx[8]
+variable        eele      equal c_mbx[9]
+variable        etot      equal c_mbx[10]
+
+fix             1 all npt temp 298 298 100 iso 1.0 1.0 1000
+
+velocity        all create 298 428459 rot yes dist gaussian
+velocity        all zero linear
+velocity        all zero angular
+
+thermo_style    custom step time temp etotal ke pe enthalpy density lx ly lz vol pxx pyy pzz press
+thermo          1
+thermo_modify   flush yes
+
+fix             thermo_print all print 1 "$(step) $(0.001*time) $(temp) $(etotal) $(ke) $(pe) $(enthalpy) $(density) $(lx) $(ly) $(lz) $(vol) $(pxx) $(pyy) $(pzz) $(press)" file thermo.out screen no title "# step time temp etotal ke pe enthalpy density lx ly lz vol pxx pyy pzz press"
+fix             verbose_mbx_thermo_print all print 1 "$(step) $(0.001*time) $(temp) $(etotal) $(ke) $(pe) $(enthalpy) $(density) $(lx) $(ly) $(lz) $(vol) $(pxx) $(pyy) $(pzz) $(press) $(v_e1bpip) $(v_e2bpip) $(v_e3bpip) $(v_e4bpip) $(v_edisp) $(v_ebuck) $(v_eperm) $(v_eind) $(v_eele) $(v_etot)" file mbx_thermo.out screen no title "# step time temp etotal ke pe enthalpy density lx ly lz vol pxx pyy pzz press e1bpip e2bpip e3bpip e4bpip edisp ebuck eperm eind eele etot"
+
+dump            1 all custom 1 dump.lammpstrj id mol type q x y z
+dump_modify     1 append no sort id format line "%d %d %d %8.4f %15.8f %15.8f %15.8f"
+
+restart         10 restart.1 restart.2
+
+run             100  upto
```

Comment:
The upto keyword is redundant here and usually not used in LAMMPS unless required.
```suggestion
run             100
```